JakeUni95/interview-questions-frontend
# Added validation to my project

If you do not select any skills an error message will appear telling to you select a skill.
You can no longer add an unknown skill into the URL.

# How to test

Run CMS locally.
Make sure that frontend files are built by running npm run build.
Run frontend locally.
Click 'Save and continue' without selecting any skills.
